test/server/cards/03_TWI/leaders/MaulARivalInDarkness.spec.ts

```diff
@@ -0,0 +1,59 @@
+describe('Maul, A Rival In Darkness', function() {
+    integration(function(contextRef) {
+        describe('Maul, A Rival In Darkness\'s undeployed leader ability', function() {
+            beforeEach(function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['death-star-stormtrooper', 'scout-bike-pursuer'],
+                        leader: 'maul#a-rival-in-darkness',
+                    },
+                    player2: {
+                        groundArena: ['specforce-soldier']
+                    },
+                });
+            });
+
+            it('should attack with a unit and it gains Overwhelm', function() {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.maulARivalInDarkness);
+                context.player1.clickPrompt('Attack with a unit. It gains Overwhelm for this attack');
+
+                context.player1.clickCard(context.deathStarStormtrooper);
```

Comment:
Before this, need to check the set of units that can be chosen for the attack using `.toBeAbleToSelectExactly()`